grpc-context-utils/src/main/java/org/hypertrace/core/grpcutils/context/FastUUIDGenerator.java

```diff
@@ -0,0 +1,25 @@
+package org.hypertrace.core.grpcutils.context;
+
+import java.util.UUID;
+import java.util.concurrent.ThreadLocalRandom;
+
+public class FastUUIDGenerator {
```

Comment:
Does this need to be public? Doesn't need a new release, but next time you're in here can you move this to package private since it's an internal impl detail.

Reply:
Initially thought of it to our own lib so it can be used. But, I think, we can make it private, anywhere else we can use the third-party library that is already there in most of the projects.

Reply:
@laxmanchekka @aaron-steinfeld https://github.com/hypertrace/java-grpc-utils/pull/50/files